lib/active_flag/definition.rb

```diff
@@ -33,16 +33,17 @@ def unset_all!(key)
     end
 
     def to_i(arg)
+      arg = arg.uniq
```

Comment:
This line has to come after the next line

Reply:
Fixed it.